roads_system/RoadRenderer.gd

```diff
@@ -166,8 +90,8 @@ func draw_intersection(intersection: RoadIntersection):
 			else:
 				direction = ((intersection.direction_to(connection.start_position) + connection.end_position.direction_to(intersection))).normalized()
 			var start_point = intersection.position
-			if intersection.connections.size() > 1:
-				start_point += direction * (connection.road_network_info.length + ((connection.road_network_info.width)/2) + (pow(intersection.connections.size(), connection.road_network_info.curvature)/2.0))
+			if intersection.connections.size() > 2:				# <-- used to be 1
+				start_point += direction * connection.road_network_info.width*1.5
```

Comment:
How could we work this out? a linear scaling factor wouldn't be great right?

Reply:
I don't know. If it's based on the road size, then I'm not sure why it wouldn't just be some factor of the road width. I mostly changed this because it seemed too large to me (especially after seeing that other game), but I couldn't work out what the code was doing in order to know how to change the parameters. :)  I also don't think you want to make the intersection larger based on the number of connections. Otherwise, it will just keep growing and growing and offsetting the roads as it does.

Reply:
The problem is, the intersection drawer kind of gives up drawing properly when the there're too many connections, also I had noticed this on other games too, the offset increases as you add more and more connections to the node.